aiohttp/web_urldispatcher.py

```diff
@@ -573,7 +573,7 @@ def __iter__(self):
         return ret
 
     def _directory_as_html(self, filepath):
-        "returns directory's index as html"
+        """returns directory's index as html"""
```

Comment:
Since you already touch docstring, please follow PEP257: `"""Render directory index as html."""`